frontend/src/hooks/useResultHandler.ts

```diff
@@ -74,24 +48,10 @@ const useResultHandler = ({ session, participant, onNext, state }: UseResultHand
                 result: mergedResults,
```

Comment:
Why not simply `result`? We don't need to assign it to the `mergedResults` variable anymore, or do we?

Reply:
Yes, good catch